src/unitxt/formats.py

```diff
@@ -131,7 +131,7 @@ class SystemFormat(BaseFormat):
 
     """
 
-    demo_format: str = "{source}\\N{target_prefix}{target}\n\n"  #  example: "User: {source}\nAgent: {target}\n\n"
+    demo_format: str = "{instruction}\\N{source}\\N{target_prefix}{target}\n\n"  #  example: "User: {source}\nAgent: {target}\n\n"
```

Comment:
I think we don't want to make this the default (so the empty prompt will have the instruction in the demo).   [ This is what causes the tests to fail].  We should do it, only in specific chat prompts.